labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift

```diff
@@ -13,11 +13,13 @@ class ProfileDetailViewController: UIViewController {
     // MARK: - Properties and Outlets
     
     @IBOutlet weak var avatarImageView: UIImageView!
-    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var firstNameLabel: UILabel!
+    @IBOutlet weak var lastNameLabel: UILabel!
     @IBOutlet weak var emailLabel: UILabel!
     
```

Comment:
The organization is clearly grouped mostly by type here. The imageView and stackView throw me off and seem random. Maybe they could be in their own sections or something if group by type is the goal here.

Reply:
I normally try to group outlets in the order they appear on the page from left->right, top->bottom and all in one section. Happy to change it so its grouped by type if that makes more sense to you

Reply:
as far as the stackview, it's listed with the views contained in it

Reply:
Ah okay I see it, that’s actually a more common way to do it. And then it looks like things are grouped based on proximity. Good as is then.